<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMnemonicAddWindow</class>
 <widget class="QDialog" name="MainMnemonicAddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Mnemonic Add Window</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_mainMnemonicName">
     <property name="text">
      <string>Main mnemonic name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_mainMnemonicName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_description">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_description"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_unitName">
     <property name="text">
      <string>Unit name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_unitName"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_typeOfUnit">
     <property name="text">
      <string>Type Of Unit</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_typeOfUnit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_sensorName">
     <property name="text">
      <string>Sensor name</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_sensorName"/>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="button_clean">
       <property name="text">
        <string>Clean</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_input">
       <property name="text">
        <string>Input</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
